src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -380,7 +392,10 @@ private Drawable readBitmapDrawable(JSONObject description, List<String> assetsL
             }
 
             final Drawable ret = new BitmapDrawable(Resources.getSystem(), image);
-            ret.setBounds(left, top, right, bottom);
+            if (useBounds) {
+                ret.setBounds(left, top, right, bottom);
```

Comment:
I agree, but I wanted to catch JS errors before I pulled and cached the image. The call to getImage does a network request, and writes to disk, so I wanted to throw a JSON error before that happened.